engine/src/main/scala/cromwell/webservice/routes/CromwellApiService.scala

```diff
@@ -182,58 +184,17 @@ trait CromwellApiService extends HttpInstrumentation with MetadataRouteSupport w
-              completeResponse(StatusCodes.Created, workflowIds.toList.map(toResponse(_, workflowState)), warnings)
-            case WorkflowStoreSubmitActor.WorkflowSubmitFailed(throwable) =>
-              throwable.failRequest(StatusCodes.BadRequest, warnings)
-          }
-        case Failure(_: AskTimeoutException) if CromwellShutdown.shutdownInProgress() => serviceShuttingDownResponse
-        case Failure(e: TimeoutException) => e.failRequest(StatusCodes.ServiceUnavailable)
-        case Failure(e) => e.failRequest(StatusCodes.InternalServerError, warnings)
-      }
-    }
-
-    onComplete(materializeFormData(formData)) {
-      case Success(data) =>
-        PartialWorkflowSources.fromSubmitRoute(data, allowNoInputs = isSingleSubmission) match {
-          case Success(workflowSourceFiles) if isSingleSubmission && workflowSourceFiles.size == 1 =>
-            val warnings = workflowSourceFiles.flatMap(_.warnings)
-            askSubmit(WorkflowStoreActor.SubmitWorkflow(workflowSourceFiles.head), warnings, getWorkflowState(workflowSourceFiles.head.workflowOnHold))
-          // Catches the case where someone has gone through the single submission endpoint w/ more than one workflow
-          case Success(workflowSourceFiles) if isSingleSubmission =>
-            val warnings = workflowSourceFiles.flatMap(_.warnings)
-            val e = new IllegalArgumentException("To submit more than one workflow at a time, use the batch endpoint.")
-            e.failRequest(StatusCodes.BadRequest, warnings)
-          case Success(workflowSourceFiles) =>
-            val warnings = workflowSourceFiles.flatMap(_.warnings)
-            askSubmit(
-              WorkflowStoreActor.BatchSubmitWorkflows(NonEmptyList.fromListUnsafe(workflowSourceFiles.toList)),
-              warnings, getWorkflowState(workflowSourceFiles.head.workflowOnHold))
-          case Failure(t) => t.failRequest(StatusCodes.BadRequest)
-        }
-      case Failure(e: TimeoutException) => e.failRequest(StatusCodes.ServiceUnavailable)
-      case Failure(e) => e.failRequest(StatusCodes.InternalServerError)
-    }
-  }
+  //  def standardSuccessHandler: PartialFunction[WorkflowStoreSubmitActorResponse, Route] = {
+  //    case WorkflowStoreSubmitActor.WorkflowSubmittedToStore(workflowId, _) => completeResponse(StatusCodes.Created, WorkflowSubmitResponse(workflowId.toString, WorkflowSubmitted.toString), Seq.empty[String])
+  //    case WorkflowStoreSubmitActor.WorkflowsBatchSubmittedToStore(workflowIds, _) => completeResponse(StatusCodes.Created, workflowIds.toList.map(x => WorkflowSubmitResponse(x.toString, WorkflowSubmitted.toString)),  Seq.empty[String])
+  //    case WorkflowStoreSubmitActor.WorkflowSubmitFailed(throwable) => throwable.failRequest(StatusCodes.BadRequest)
+  //  }
+  //
+  //  def standardErrorHandler: PartialFunction[Throwable, Route] = {
+  //    case _: AskTimeoutException if CromwellShutdown.shutdownInProgress() => serviceShuttingDownResponse
```

Comment:
Is this commented part a planned future addition?

Reply:
Also an artifact and gone!